townhall/tests/endpoint_tests/test_organization_endpoint.py

```diff
@@ -1,9 +0,0 @@
-import pytest
```

Comment:
flake8 does not allow unused imports

Reply:
Good practice for the team not to leave unused imports :D